src/github.com/matrix-org/go-neb/services/rssbot/rssbot.go

```diff
@@ -34,6 +35,16 @@ var (
 
 const minPollingIntervalSeconds = 60 * 5 // 5 min (News feeds can be genuinely spammy)
 
+// includeRules contains the rules for including or excluding a feed item.
+type includeRules struct {
+	// Author is a list of strings that the author name must match or not match.
+	Author []string `json:"author"`
```

Comment:
Also please state what happens in the empty set case of`[]`. If I have `include: { author: [] }` does that filter everything out which has an author tag? We likely will want to treat empty values as not-set, so no filtering is applied.

Reply:
Indeed, that's what already happened (empty == unset). Added this to the docs.